roles/vpn/templates/openssl-server-certificate.cnf.j2

```diff
@@ -39,19 +39,14 @@ distinguished_name = req_distinguished_name
 
 [ req_distinguished_name ]
 countryName = Country Name (2 letter code)
-countryName_default = {{ openvpn_key_country }}
```

Comment:
My mistake. I comingled #525 in this PR. I'll fix this.

On 03/11, Yannik wrote:
> > @@ -39,19 +39,14 @@ distinguished_name = req_distinguished_name
> >  
> >  [ req_distinguished_name ]
> >  countryName = Country Name (2 letter code)
> > -countryName_default = {{ openvpn_key_country }}
> 
> What happened here?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sovereign/sovereign/pull/526/files#r55835250